Makefile

```diff
@@ -15,8 +15,8 @@ endif
 
 # Image URL to use all building/pushing image targets
 IMG ?= flanksource/karina:latest
-# Produce CRDs that work back to Kubernetes 1.11 (no version conversion)
-CRD_OPTIONS ?= "crd:trivialVersions=true"
+# Produce CRDs with v1
+CRD_OPTIONS ?= "crd:crdVersions=v1"
```

Comment:
I know I said we could include the controller-gen changes in this PR, but it probably makes sense to split it out so it can merge faster. 

Even when including in the same PR, we would want to split `feat:` and `refactor:` commits